src/lib/hooks.js

```diff
@@ -11,6 +11,9 @@ const createOrUpdateGuest = async (guest) => {
 	const { token } = guest;
 	token && await store.setState({ token });
 	token && await loadConfig();
+	console.log('Data');
+	console.log(guest);
```

Comment:
Do we want to keep these log points or are they debug?

Reply:
Removing these.

Reply:
@ear-dev Are you sure you are looking at the latest commit? These are removed in the [45f8b80](https://github.com/WideChat/Rocket.Chat.Livechat/pull/127/commits/45f8b80ab3f7a94cfcd41a94b79b62589fe71de3)

Reply:
gottit......  sorry about that.